programs/client/Client.cpp

```diff
@@ -842,6 +842,7 @@ void Client::addOptions(OptionsDescription & options_description)
 
         ("no-warnings", "disable warnings when client connects to server")
         ("fake-drop", "Ignore all DROP queries, should be used only for testing")
+        ("accept-certificate", "Ignore certificate verification errors, equal to config parameters openSSL.client.invalidCertificateHandler.name=AcceptCertificateHandler and openSSL.client.verificationMode=none")
```

Comment:
I suggest to name this option in a way that indicate that it is not a usual case. "accept-certificate" looks like it a good option if you want to enable secure connection, so good and clear that there is no point to read the description, just turn it on. 
The description is fine, but the user needs some hint to read it.

I think "accept-invalid-certificate" should work. 3 words for unusual case is fair.